packages/edit-site/src/components/sync-state-with-url/use-sync-path-with-url.js

```diff
@@ -41,107 +45,145 @@ export function getPathFromURL( urlParams ) {
 	return path;
 }
 
-function isSubset( subset, superset ) {
-	return Object.entries( subset ).every( ( [ key, value ] ) => {
-		return superset[ key ] === value;
-	} );
+function getParamsFromPath( path, params ) {
```

Comment:
This is better than what we had but still feels hacky. I'd love to be able to use the same `useLocation/useParams/useRouter` to retrieve the params from the url, regardless of where I am (sidebar, content...)